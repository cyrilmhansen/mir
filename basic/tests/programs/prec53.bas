10 REM constant with >53 bits of precision exercises long double printing
20 REM double rounding differs from extended precision
30 PRINT 0.123456789012345678901
