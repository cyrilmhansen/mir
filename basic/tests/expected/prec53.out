0.123456789012346
